X/XGBoost/build_tarballs.jl

```diff
@@ -4,9 +4,8 @@ using BinaryBuilder, Pkg
 name = "XGBoost"
 version = v"0.82"
 sources = [
-    "https://github.com/dmlc/xgboost.git"=>
-    "3f83dcd50286d7c8d22e552942bd6572547c32b9",
-    "./bundled",
+    GitSource("https://github.com/dmlc/xgboost.git","bf3241368256ddd010d30d98ffc8a0a005f166e9"),
```

Comment:
@ViralBShah where did you get this git-sha from? The previous value is the tagged `0.8.2` version of XGBoost, but I cant find `bf3241` sha in https://github.com/dmlc/xgboost

Reply:
Looks like it's in the `release_0.82` branch: https://github.com/dmlc/xgboost/commit/bf3241368256ddd010d30d98ffc8a0a005f166e9